src/app/main/main-window/sidebar/sidebar.component.ts

```diff
@@ -96,7 +96,7 @@ export class SidebarComponent {
     }
   }
 
-  logout() {
+  disconnect() {
```

Comment:
```suggestion
  disconnect(): void {
```

I know that it's not related to this PR but we try to write always the return type of the function :+1: